app/src/main/res/xml/preferences_user_interface.xml

```diff
@@ -92,5 +92,12 @@
                 android:key="prefSwipe"
                 android:summary="@string/swipeactions_summary"
                 android:title="@string/swipeactions_label"/>
+        <de.danoeh.antennapod.preferences.MaterialListPreference
+                android:entryValues="@array/globalNewEpisodesActionValues"
+                android:entries="@array/globalNewEpisodesActionItems"
+                android:key="prefNewEpisodesAction"
+                android:title="@string/pref_new_episodes_action_title"
+                android:summary="@string/pref_new_episodes_action_sum"
+                android:defaultValue="add_to_inbox"/>
```

Comment:
> Does this option belong in the "User Interface" menu, or should it be moved to "Network"?

I don't think it fits well into the "user interface" settings. "Network" seems a better fit to me.

Reply:
I have moved it into the network settings. It seemed to fit nicely in the "automation" group since this is an action that should be taken automatically for new episodes.